vendor/vendor.json

```diff
@@ -3913,6 +3863,14 @@
 			"revision": "5500ca0de0dab231b02aedabac095d43a59f31d2",
 			"revisionTime": "2019-05-22T20:56:50Z"
 		},
+		{
+			"checksumSHA1": "cFcwn0Zxefithm9Q9DioRNcGbqg=",
+			"path": "github.com/hashicorp/go-multierror",
```

Comment:
Don't we already use an other lib for multierror? @andrewkroh

Reply:
@ruflin  we do, but can we remove it in a followup PRs I don't want to mess around with refactor and merging with master.